Configurations/unix-Makefile.tmpl

```diff
@@ -1206,6 +1245,20 @@ tags TAGS: FORCE
 	-ctags -R .
 	-etags `find . -name '*.[ch]' -o -name '*.pm'`
 
+{- output_off() if $disabled{fips}; "" -}
+fips-checksums: generate_fips_sources
+	if which unifdef > /dev/null; then \
+	    cat $(SRCDIR)/providers/fips.module.sources \
+	       | (cd $(SRCDIR); xargs ./util/fips-checksums.sh ) \
+	       > $(SRCDIR)/providers/fips-sources.checksums \
+	    && openssl sha256 -hex \
```

Comment:
Could we do something like `(cd $(SRCDIR/providers); openssl sha256 -hex fips-sources.checksums >fips.checksum); \` here to get something else than stdin in the fips.checksum file?

Reply:
Sure

Reply:
Done